.gitignore

```diff
@@ -1,2 +1,3 @@
+/dist/
```

Comment:
do we have an npmignore? Because otherwise, npm would ignore the dist folder as well.